tasks/publish-to-cgw/publish-to-cgw.yaml

```diff
@@ -70,9 +70,10 @@ spec:
         import subprocess
 
         DATA_FILE = "$(workspaces.data.path)/$(params.dataPath)"
+        CONTENT_DIR = "$(workspaces.data.path)/$(params.contentDir)"
```

Comment:
Why this change? It makes the task no longer safe against overwrites if parallel pipelineruns are running

Reply:
Previously we had the following:
```
WORKSPACE_DIR = "/workspace/data/{sub_dir}"
$(params.contentDir) = "{sub_dir}/binaries"
CONTENT_DIR = f"{WORKSPACE_DIR}/$(params.contentDir)" 
```
This change is made so we don't have the extra sub_dir passed from extract-binaries-from-image

Reply:
I mean to say that `WORKSPACE_DIR = os.path.dirname(DATA_FILE)` translates to `WORKSPACE_DIR = "/workspace/data/{sub_dir}"` 

I agree dirname approach is best.
The reason for this small change is that we were ending up with `workspace/data/sub_dir/sub_dir/binaries`
notice the duplicate sub_dir

I see two options to make this work.

1.  extract-binaries-to-image results just binaries instead of subdir/binaries
2.  get basename of  `$(params.contentDir)` and then append it to  WORKSPACE_DIR

Reply:
So are you saying `$(params.contentDir)` already contains the pipelinerun specific subdir?

Reply:
yea, correct.